### PR TITLE
Made the first steps towards integration with twitter stream.

### PR DESCRIPTION
**Pull request by AlpineMutts**
- NodeJS server for handling twitter stream api.
- Allow Ajax calls to NodeJS server.
- Create a new ball on each occurrence of a new tweet.

Todo: lots of refactoring of the code.

### CODE REVIEW

File: js/time.js [hunk 1]
```diff
@@ -26,6 +30,32 @@ var subscribeToClockObservable = function () {
 };
 
 /**
+ * Subscribe to the clock subscription.
+ */
+var subscribeToTweets = function () {
+  clock.filter(function(t){
```
Comment (JoopAue) [inline js/time.js]: clock is renamed to clockObservable

**Comment by JoopAue** [comment]
Would you like this to be merged? So that we can refactor from master?
Or do you want time to refactor it in this branch?

**Comment by AlpineMutts** [comment]
Yeah sorry I should've mentioned that, sorry. I will refactor it in this branch.

**Comment by JoopAue** [comment]
missing in package.json
- cookie-parser 
- connect-flash
- passport

**Comment by AlpineMutts** [comment]
Feel free to contribute on these branches.

**Comment by AlpineMutts** [comment]
:+1:

**Comment by AlpineMutts** [comment]
So I guess, I'll throw this out...

**Pull request closed without merging**